SC_redesign/Crop_and_Soil/soil/layer_data.py

```diff
@@ -409,14 +484,14 @@ def determine_soil_phosphorus_concentration(labile_phosphorus: float, bulk_densi
         return soil_phosphorus_mass_in_mg / soil_mass_in_kg
 
     @staticmethod
-    def determine_soil_phosphorus_area_density(labile_phosphorus: float, bulk_density: float,
-                                               layer_thickness: float, field_size: float) -> float:
-        """Converts a concentration of soil from (mg / kg soil) to (kg / ha)
+    def determine_soil_nutrient_area_density(nutrient_concentration: float, bulk_density: float,
+                                             layer_thickness: float, field_size: float) -> float:
+        """Converts a concentration of nutrients in the soil from (mg / kg soil) to (kg / ha)
 
         Parameters
         ----------
-        labile_phosphorus : float
-            Labile phosphorus content of this soil layer (mg phosphorous / kg soil)
+        nutrient_concentration : float
+            Nutrient concentration of this soil layer (mg / kg soil)
```

Comment:
since the description already mentioned in the soil, maybe we can skip the soil in the unit?
```suggestion
            Nutrient concentration of this soil layer (mg / kg)
```

Reply:
You raise a good point, this was a little redundant. I did sort of the opposite though, I removed the specifics from the sentence description and kept the specific units in the parameter descriptions.